<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1811</width>
    <height>938</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>781</width>
     <height>931</height>
    </rect>
   </property>
   <property name="title">
    <string>选择区</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>59</y>
      <width>771</width>
      <height>871</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </widget>
   <widget class="QLineEdit" name="txt_path">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>文件路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_choose">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>8</y>
     <width>1001</width>
     <height>921</height>
    </rect>
   </property>
   <property name="title">
    <string>可视区</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>461</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_AF"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>461</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="VLBotoom_AF">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>441</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_SHG"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>450</y>
      <width>441</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="VLBotoom_SHG">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
    </layout>
   </widget>
   <widget class="QLabel" name="selectLable">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>AF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>SGH</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
